### ISSUE TITLE
ImportError: cannot import name '_CAPACITY'

### ISSUE THREAD

**Issue by afshar-mohammad** [smitchell556/flask-cuttlepool#11]
Just installed the latest flask-cuttlepool and imported it:

`from flask_cuttlepool import FlaskCuttlePool`

and got this error on python 3.6 (cuttlepool-0.9.1 flask-cuttlepool-0.2.0):

```
from flask_cuttlepool import FlaskCuttlePool
  File "/usr/local/lib/python3.6/dist-packages/flask_cuttlepool.py", line 24, in <module>
    from cuttlepool import (CAPACITY as _CAPACITY, OVERFLOW as _OVERFLOW,
ImportError: cannot import name 'CAPACITY'
[2020-05-10 22:49:30 +0200] [21136] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/flask_cuttlepool.py", line 21, in <module>
    from cuttlepool import _CAPACITY, _OVERFLOW, _TIMEOUT
ImportError: cannot import name '_CAPACITY'
```

**Comment by spenceforce** [comment]
There have been changes made to cuttlepool that break flask-cuttlepool. For now, I will update setup.py to use cuttlepool v0.7.0 explicitly instead of >=0.6.0 which will fix this problem. Moving forward, flask-cuttlepool will need to be updated itself to use the most recent version of cuttlepool, which is the recommended one to use.

**Comment by spenceforce** [comment]
Resolved 73bb0852d2b4ccfedde8d5c46ce939868f0bceb5 with version 0.2.1

**Comment by spenceforce** [comment]
@pazis  it just occurred to me that by using flask-cuttlepool, you may run into a race condition present in cuttlepool<=0.9.0. For the time being while I rework flask-cuttlepool to use cuttlepool>=0.9.1, I recommend using the latest version of [cuttlepool](https://github.com/smitchell556/cuttlepool) as a global object until flask-cuttlepool is compatible with it. It’s a trade off between convenience vs thread safety.
